src/test/ui/tuple/tuple-struct-fields/test2.stderr

```diff
@@ -2,12 +2,13 @@ error: expected one of `)` or `,`, found `(`
   --> $DIR/test2.rs:5:26
    |
 LL |         struct S3(pub $t ());
-   |                         -^ expected one of `)` or `,`
-   |                         |
-   |                         help: missing `,`
+   |                          ^ expected one of `)` or `,`
```

Comment:
This diagnostic is probably using `prev_token.span` which previously pointed to `$t` and now points to `)` in `(foo)`.

If `Invisible` delimiters weren't dropped, then this would continue working without changes, because the closing `Invisible` delimiters would be used as `prev_token`, and it inherits its span from `$t`.